src/import-data/ImportData.test.ts

```diff
@@ -445,8 +445,8 @@ describe('ImportData', () => {
   it.each([
     { queryParams: { format: 'pfb' }, expectedErrorMessage: 'A URL is required' },
     {
-      queryParams: { format: 'tdrexport', snapshotId: '00001111-2222-3333-aaaa-bbbbccccdddd' },
-      expectedErrorMessage: 'A manifest URL is required',
+      queryParams: { format: 'tdrexport' },
```

Comment:
is there still a test for when a manifest url is required (in the previous use case)?